lib/chart/gauge.ex

```diff
@@ -1,11 +1,20 @@
 defmodule Chart.Gauge do
   @moduledoc """
-  Gauge graph
+
+  ![Gauge](docs/gauge.png)
+
+  An appearance of gauge graph can be change via `config_keywords` (see below `setup` function)
```

Comment:
[Make reference to function as html anchor.](https://hexdocs.pm/elixir/writing-documentation.html#recommendations)

Reply:
thx, I always forget it